lib/win-rebase.js

```diff
@@ -1,8 +1,17 @@
 var re = /^\s*("*)([^"]*?\b(?:node|iojs)(?:\.exe)?)("*)( |$)/
+var npmre = /^\s*("*)([^"]*?\b(?:npm))("*)( |$)/
+var path_ = require('path')
+if (path_.win32) path_ = path_.win32
 
-module.exports = function (path, rebase) {
+module.exports = function (path, rebase, whichOrUndefined) {
```

Comment:
Idk… I’m not too proud of this way either, but right now I don’t see any way in which having this code here + testing it doesn’t get even messier with a “proper”, module-based solution.